### DISCUSSION TITLE
witch 'lua-nginx-module' should i use?

### DISCUSSION THREAD

**Discussion by xiongshiyan** [apache/skywalking#8283] [Q&A]
when i use newest lua-nginx-module, my nginx canot start up。i want to ask which lua-nginx-module should i use or it's other problem?
```
--prefix=/home/djc/install/nginx/nginx-1.20.1 \
--with-http_stub_status_module \
--with-http_gzip_static_module \
--with-http_ssl_module \
--add-module=../ngx_devel_kit-0.3.1 \
--add-module=../lua-nginx-module-master \
--with-ld-opt=-Wl,-rpath,/usr/local/LuaJIT/lib
```
![image](https://user-images.githubusercontent.com/39974039/145556291-3a2104fb-9919-4dd2-987c-ebff4312716f.png)

**Comment by wu-sheng** [comment]
We require OpenResty rather than pure Nginx.
